src/BinSkim.Sdk/BinaryAnalyzerContext.cs

```diff
@@ -86,8 +66,6 @@ public override string MimeType
 
         public override bool AnalysisComplete { get; set; }
 
-        public override DefaultTraces Traces { get; set; }
```

Comment:
>Traces [](http://example.com/codeflow?start=38&length=6)

This property and ForceOverwrite have been pushed lower into the stack, they are standard on the SDK base context classes.